<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoiceRceivingMethod</class>
 <widget class="QDialog" name="ChoiceRceivingMethod">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChoiceRceivingMethod</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Choose the method through which to recive the servers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="wtsRadio">
     <property name="text">
      <string>WTS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="ldapRadio">
     <property name="text">
      <string>LDAP</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="currentRadio">
     <property name="text">
      <string>Current server (localhost)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="userRadio">
     <property name="text">
      <string>Uset Input</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
